### DISCUSSION TITLE
Choosing rotation angle from input list

### DISCUSSION THREAD

**Discussion by kvenkman** [kornia/kornia#2974] [Potential Issue]
Hello, I was interested in applying a rotational transform to my input data during training. Specifically, I was interested in rotating my images by randomly choosing a value from the list [0, 90, 180, 270]. Is this possible in Kornia? 

Alternatively, can I contribute a new augmentation (e.g. called RandomDiscreteRotation) that will implement this functionality?

**Comment by edgarriba** [comment]
i believe that with [`RandomRotation`](https://kornia.readthedocs.io/en/stable/augmentation.module.html#kornia.augmentation.RandomRotation) you can pass a list of rotations to select. Otherwise, for sure feel free to contribute

**Comment by kvenkman** [comment]
The following code: 

```
from kornia.augmentation import RandomRotation
aug = RandomRotation([10, 20, 30])
```

throws the error:
`TypeError: degrees should be a tensor with length 2 whose values between (-360.0, 360.0). Got tensor([10., 20., 30.]).`

which leads me to think that `RandomRotation` does not accept a list of rotations to choose from. I can implement this and raise PR if that sounds good.

**Reply by edgarriba** [reply]
sounds good to me. Any thoughts @johnnv1 @shijianjian ?

**Reply by johnnv1** [reply]
sounds good to me
